### ISSUE TITLE
laji

### ISSUE THREAD

**Issue by sherrycherish** [airbnb/react-sketchapp#71]
```
npm run render       ~/Downloads/basic-setup/node_modules 127 ↵

> basic-setup@1.0.0 render /Users/sherry/Downloads/basic-setup
> skpm build --watch --run

/Users/sherry/Downloads/basic-setup/node_modules/skpm/lib/utils/sketchCommandPlugin.js:10
  const { bundleURL, commandIdentifier } = self.options
        ^

SyntaxError: Unexpected token {
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/sherry/Downloads/basic-setup/node_modules/skpm/lib/utils/webpackConfig.js:5:27)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)

npm ERR! Darwin 16.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "render"
npm ERR! node v4.5.0
npm ERR! npm  v2.15.9
npm ERR! code ELIFECYCLE
npm ERR! basic-setup@1.0.0 render: `skpm build --watch --run`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the basic-setup@1.0.0 render script 'skpm build --watch --run'.
npm ERR! This is most likely a problem with the basic-setup package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     skpm build --watch --run
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs basic-setup
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!
npm ERR!     npm owner ls basic-setup
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/sherry/Downloads/basic-setup/node_modules/npm-debug.log
```

**Comment by ljharb** [comment]
You're using node 4, which doesn't understand destructuring. skpm might require node 6 or later (which I'd consider a bug in skpm; everything should work on node 4+ at a minimum) - can you try it on node 6 or 7?

**Comment by sherrycherish** [comment]
@ljharb  problem solved. 
thx 😄 .

**Comment by mathieudutour** [comment]
Fixed it in `skpm@0.9.7`.
